### ISSUE TITLE
plot_contour

### ISSUE THREAD

**Issue by z7ye** [optuna/optuna#1803]
## Environment

- Optuna version: 2.0.0
- Python version:3.6.9
- OS:LINUX
- (Optional) Other libraries and their versions:

## Error messages, stack traces, or logs
in version 2.0.0 the code for contour plot use "max_value - min_value", in the case of categorical features, this will give an error. 
```
study = <optuna.study.Study object at 0x7f7af7a4e0b8>, params = None

      def _get_contour_plot(study: Study, params: Optional[List[str]] = None) -> "go.Figure":

          layout = go.Layout(title="Contour Plot")
          trials = [trial for trial in study.trials if trial.state == TrialState.COMPLETE]
          if len(trials) == 0:
              _logger.warning("Your study does not have any completed trials.")
              return go.Figure(data=[], layout=layout)
          all_params = {p_name for t in trials for p_name in t.params.keys()}
          if params is None:
              sorted_params = sorted(list(all_params))
          elif len(params) <= 1:
              _logger.warning("The length of params must be greater than 1.")
              return go.Figure(data=[], layout=layout)
          else:
 >               padding = (max_value - min_value) * padding_ratio
  E               TypeError: unsupported operand type(s) for -: 'str' and 'str'
```
In version 1.9.0, 
```
param_values_range = dict()
    for p_name in sorted_params:
        values = [t.params[p_name] for t in trials if p_name in t.params]
        param_values_range[p_name] = (min(values), max(values))
```

**Comment by ytknzw** [comment]
I think this can be fixed with any numerical conversion like in ``_plot_parallel_coordinate.py``:
```
        try:
            tuple(map(float, values))
        except (TypeError, ValueError):
            vocab = defaultdict(lambda: len(vocab))  # type: DefaultDict[str, int]
            values = [vocab[v] for v in values]
```

**Comment by sile** [comment]
Fixed by #1819
